### ISSUE TITLE
After 3.0 update I have a problem with the config file on WSL

### ISSUE THREAD

**Issue by xasweil** [dylanaraps/neofetch#630]
## Description
After today's update ceased to work the following config options: "progress bar colors", "progress bar length", "progress bar character", "birthday time". Also appeared an empty space after the output of the script.

#### Neofetch version
3.0
## Screenshot
Problematic areas are highlighted in pink.

![screenshot of preview](https://i.imgur.com/hMJdnn6.jpg)
## Verbose log
http://pastebin.com/B1SQpHUx

**Comment by dylanaraps** [comment]
Does the issue still occur with a new config file?

**Comment by dylanaraps** [comment]
Try out the latest master and let me know if the issues still persist.

**Comment by xasweil** [comment]
Yes, using a new config file is not present more problems, but below the script is still a lot of empty space.

![screenshot of preview](https://i.imgur.com/9C6t9gn.png)

**Comment by dylanaraps** [comment]
Is this screenshot with the new config?

**Comment by iraquitan** [comment]
I'm having a similar problem. I'm now using the same config file as in master, but the output seems to be broken on the shell command. The output is cropped.
<img width="879" alt="screen shot 2017-01-24 at 08 19 34" src="https://cloud.githubusercontent.com/assets/3467229/22245293/01323e42-e20e-11e6-9f51-c7d7ce7732c4.png">

if I comment the shell command on config, the output is not cropped:
<img width="852" alt="screen shot 2017-01-24 at 08 22 21" src="https://cloud.githubusercontent.com/assets/3467229/22245379/5b91a954-e20e-11e6-891e-da67ef054f86.png">

## Verbose log
The verbose log is found here: http://pastebin.com/sifHL6EJ

## Config file
And the config file used here: http://pastebin.com/K2YF33jv

**Comment by dylanaraps** [comment]
@iraquitan, what shell are you using? `echo $SHELL`

**Comment by xasweil** [comment]
> Is this screenshot with the new config?

Yes

> Is this screenshot with the latest git master?

Yes

**Comment by dylanaraps** [comment]
Can you send me your config file?

**Comment by xasweil** [comment]
Uploaded it here: http://pastebin.com/rNzr3QXa
I also noticed what does not work "ascii distro" section.

**Comment by dylanaraps** [comment]
I can't reproduce the issue using your config file.

![2017-01-24-225130_460x320_scrot](https://cloud.githubusercontent.com/assets/6799467/22246249/fadbf428-e287-11e6-8e83-0be6718afe7a.png)

**Comment by iraquitan** [comment]
@dylanaraps I'm using `/bin/zsh`. But I'm not using the latest git master. I installed through homebrew.

**Comment by dylanaraps** [comment]
@iraquitan, can you open a new issue? I don't think yours is related to @xasweil's and it'll make it easier to communicate. :)

**Comment by xasweil** [comment]
I think I found the problem, ConEmu Windows console emulator displays a blank space after the script, but the standard command line displays fine.

![screenshot of preview](https://i.imgur.com/Xv5kU1c.png)

The only remaining problem is the broken "ascii distro" section. You can set different distro logo using my config file?

**Comment by dylanaraps** [comment]
`ascii_distro` works for me using your config file. Does setting ascii_distro at launch (`--ascii_distro gentoo`) work on your system?

**Comment by xasweil** [comment]
No, it's still displayed Windows logo.

**Comment by dylanaraps** [comment]
I can reproduce this by forcing my OS to Windows 10. I'm working on a fix now :+1:

**Comment by dylanaraps** [comment]
Try out the latest master and let me know if this is fixed. :)

**Comment by dylanaraps** [comment]
Anytime :)
